javascript/javascript1/week2/smart-ease.js

```diff
@@ -0,0 +1,180 @@
+// FLIGHT BOOKING FULLNAME FUNCTION
+
+function getFullname(firstname, surname, useFormalName, gender) {
+    if (firstname && surname && useFormalName == true && gender) {
+        if (gender == "male") {
+            return "Lord " + "Mr." + firstname + ' ' + surname;
+        }
+        else {
+            return "Ms." + firstname + ' ' + surname;
+        }
+    }
+    else if (firstname && surname && useFormalName == true) {
+        return "Lord " + firstname + ' ' + surname;
+    }
+    else if (firstname && surname) {
+        return firstname + ' ' + surname;
+    }
+    else if (firstname || surname) {
+        return "Enter both names";
+    }
+    else {
+        return "No name given";
+    }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2 = getFullname("Rasim", "Deniz");
+const fullname3 = getFullname("Daniel");
+const fullname4 = getFullname();
+const fullname5 = getFullname("Marie", "Hoff", true, "female");
+console.log(fullname1);
+console.log(fullname2);
+console.log(fullname3);
+console.log(fullname4);
+console.log(fullname5);
+
+// EVENT APPLICATION
+
+const weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+const today = new Date();
+
+function getEventWeekday(eventDay) {
+    let i = (today.getDay() + eventDay) % 7;
+    return weekDays[i];
+}
+
+console.log(getEventWeekday(7));
+console.log(getEventWeekday(2));
+
+// WEATHER WEAR
+
+function getClothesForToday(temperature) {
+    if (temperature < 0) {
+        return "Jacket";
+    }
+    else if (temperature >= 0 && temperature < 15) {
+        return "Sweater";
+    }
+    else if (temperature >= 15 && temperature < 40) {
+        return "T-shirt";
+    }
+    else {
+        return "Naked";
+    }
+}
+
+const clothesToWear = getClothesForToday(28);
+console.log(clothesToWear);
+
+// STUDENT MANAGER
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.length < 6 && studentName !== "Queen" && studentName) {
+        class07Students.push(studentName);
+        return class07Students;
```

Comment:
you don't need to return the array if you declared as global...maybe return something like "true/false" . True when you are able to add a student and false when you can't
```suggestion
return true
```